src/etc/htmldocck.py

```diff
@@ -159,11 +159,11 @@ def handle_data(self, data):
         self.__builder.data(data)
 
     def handle_entityref(self, name):
-        self.__builder.data(unichr(name2codepoint[name]))
+        self.__builder.data(chr(name2codepoint[name]))
 
     def handle_charref(self, name):
         code = int(name[1:], 16) if name.startswith(('x', 'X')) else int(name, 10)
-        self.__builder.data(unichr(code))
+        self.__builder.data(chr(code))
```

Comment:
AFAICT, this drops compatibility with python 2 here -- is there a way to avoid that?

Reply:
Oups indeed!